docs/custom-metrics-apiservice.yaml

```diff
@@ -1,13 +1,13 @@
-apiVersion: apiregistration.k8s.io/v1beta1
+apiVersion: apiregistration.k8s.io/v1
 kind: APIService
 metadata:
-  name: v1beta1.custom.metrics.k8s.io
```

Comment:
@mikkeloscar you know if there's any reason to use [`v1beta1.custom.metrics.k8s.io`](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/kube-metrics-adapter/custom-metrics-apiservice.yaml#L4)?

Reply:
Ok, so far as I could check, that's just the name and we don't refer to that anywhere else.

Reply:
At least in my local testing today I am using v1 and I haven't encountered any issues so far. I'll let you know if anything pops up.

Reply:
I think it maps to the versions defined here: https://github.com/kubernetes/metrics/tree/master/pkg/apis/custom_metrics so we should not update this one otherwise it exposes an unexpected API as v1 doesn't exist.

Reply:
makes more sense now. So we can follow [these examples](https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/0ca2b1909cdc45259654a4e06cac40f98fa1bcb0/test-adapter-deploy/testing-adapter.yaml#L108-L134)  and update just the API version of the `APIService` itself. @Miniland1333 do you mind reverting these changes?

Reply:
That makes sense and it explains the weird issue I was experiencing at the end of yesterday. Also, quick local testing indicates that metrics API v1beta2 is not a drop-in replacement as when querying the endpoint you get `Error from server (ServiceUnavailable): the server is currently unable to handle the request`
![image](https://user-images.githubusercontent.com/12994749/116286654-50ca9300-a744-11eb-9e86-946d4f32793e.png)

Would you like me to add the blocks for `v1beta2.external.metrics.k8s.io` as you suggested in [this example](https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/0ca2b1909cdc45259654a4e06cac40f98fa1bcb0/test-adapter-deploy/testing-adapter.yaml#L108-L134) or since it isn't currently working do you instead want it to be in a separate PR.

Reply:
Let's do this separately.